python/tvm/relay/op/contrib/dnnl.py

```diff
@@ -76,12 +78,79 @@ def make_pattern(with_bias=True):
         conv_out = is_op("add")(conv, bias)
     else:
         conv_out = conv
-    return is_op("nn.relu")(conv_out)
+    if with_relu:
+        return is_op("nn.relu")(conv_out)
+    return conv_out
+
+
+def make_dense_pattern(with_bias=True, with_relu=True):
+    data = wildcard()
+    weight = wildcard()
+    bias = wildcard()
+    dense = is_op("nn.dense")(data, weight)
+    if with_bias:
+        dense_out = is_op("add")(dense, bias)
+    else:
+        dense_out = dense
+    if with_relu:
+        dense_out = is_op("nn.relu")(dense_out)
+    return dense_out
 
 
 @register_pattern_table("dnnl")
 def pattern_table():
-    conv2d_bias_relu_pat = ("dnnl.conv2d_bias_relu", make_pattern(with_bias=True))
-    conv2d_relu_pat = ("dnnl.conv2d_relu", make_pattern(with_bias=False))
-    dnnl_patterns = [conv2d_bias_relu_pat, conv2d_relu_pat]
+    conv2d_bias_relu_pat = (
+        "dnnl.conv2d_bias_relu",
+        make_conv_pattern(with_bias=True, with_relu=True),
+    )
+    conv2d_bias_pat = ("dnnl.conv2d_bias", make_conv_pattern(with_bias=True, with_relu=False))
+    conv2d_relu_pat = ("dnnl.conv2d_relu", make_conv_pattern(with_bias=False, with_relu=True))
+    dense_bias_pat = ("dnnl.dense_bias", make_dense_pattern(with_bias=True, with_relu=False))
+    dnnl_patterns = [
+        conv2d_bias_relu_pat,
+        conv2d_bias_pat,
+        conv2d_relu_pat,
+        dense_bias_pat,
+    ]
     return dnnl_patterns
+
+
+def partition_for_dnnl(
+    mod,
```

Comment:
format? Keep the two args in the same line of function declaration.

Reply:
fixed. TVM has C++ and python lint, all lint has been passes.